packages/image/src/index.ts

```diff
@@ -4,15 +4,19 @@ export type { ImageLoader } from "./image-optimize";
 export * as loaders from "./image-loaders";
 import { props } from "./__generated__/image.props";
 
-// "loader" is our internal prop not intended to show up in the props panel
-const { loader, ...publicProps } = props;
+const getImageProps = (): Record<string, PropMeta> => {
+  // "loader" is our internal prop not intended to show up in the props panel
+  const { loader, ...publicProps } = props;
```

Comment:
Probably only this spread causes issue, because the other kind of spread we use in many other components.

If so, might be better to put only it in a function to make situation more clear

Reply:
Nope, looks like spread affects bundle too

Reply:
Just strange that spread here for example is not a problem https://github.com/webstudio-is/webstudio-builder/blob/317edc0d79a2dbcea0b9684a7e8f48113911a77d/packages/react-sdk/src/components/link.ws.tsx#L37

Or maybe it's a problem too